gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/rest/api/service/impl/configuration/application/registration/ClientRegistrationServiceImpl.java

```diff
@@ -333,6 +333,23 @@ public ClientRegistrationResponse register(ExecutionContext executionContext, Ne
         return registrationProviderClient.register(clientRegistrationRequest);
     }
 
+    private static ClientRegistrationRequest convert(NewApplicationEntity application) {
+        var mapped = ClientRegistrationMapper.INSTANCE.toClientRegistrationRequest(application);
+        var objectMapper = new ObjectMapper();
```

Comment:
⛔️ inject the object mapper, do not create one manually for each request. moreover, it will not use the configuration of the one configured for spring context